src/test/java/com/vinylteam/vinyl/util/RawOffersSorterTest.java

```diff
@@ -188,4 +188,88 @@ void addOffersSortingByVinylRawOffersZeroMatchesTest() {
         assertEquals(expectedOffers, actualOffers);
     }
 
+    @Test
+    void fordel() {
```

Comment:
fordel?

Reply:
oops.. yeah.. it was for del :) i forgot about that